### DISCUSSION TITLE
Web Ui

### DISCUSSION THREAD

**Discussion by Mehdi-YC** [leon-thomm/Ryven#86] [Ideas]
a web Ui would be a great addition for the project since every thing is moving to the cloud now
i use a lot of ui based ETLs
like talend and knime
buut there is no great ui for web versions like luigi and airflow
you can see the execution process but not the number of variables input and output for example 

this project had a good ui and can be used as etl with Petl or pandas

**Comment by leon-thomm** [comment]
thanks! yep, a web UI for a remote (or local) ryvencore session would be amazing. In fact, that's what I was most interested in over the past few months. I myself just don't have a strong enough web background (so far) to lay the foundations, because I would assume that most of the UI development in the future would go there, so it should be scalable

**Comment by Mehdi-YC** [comment]
Hello , well im not the best web dev , i can work with svelte and a lil bit of react and fluent with the basics (html css & js)
the problem is i never tryed to implement such a complicated UI with floatng elements , if we find someone with a solid web background i can help make this possible i have 4 y + experience with python

**Reply by dfpena** [reply]
If you want a web UI you can use https://pluricorp.com/stremecoder/. It's free and been around since 2019.

**Reply by Mehdi-YC** [reply]
o thank you for this , i'v never seen it before , and trust me i search a lot lol , have a good day

**Reply by vkuehn** [reply]
looks great as a tool but not open source thought...
